superset/assets/javascripts/explorev2/stores/fields.js

```diff
@@ -1006,15 +1013,16 @@ export const fields = {
     type: 'TextField',
     label: 'Opacity',
     default: 1,
+    validators: [v.numeric],
```

Comment:
Right, like a `rangeValidator`, I thought a bit about how to pass static configuration parameters to the validator from here and though we could probably bind things in this context, as in `v.range.bind(0, 1)`